lib/src/realm_object.dart

```diff
@@ -178,6 +178,18 @@ class RealmCoreAccessor implements RealmAccessor {
   }
 }
 
+mixin RealmEntity {
+  Realm? _realm;
+  /// The [Realm] instance this collection belongs to.
```

Comment:
realm entity is not a collection. The doc comment needs a fix. Perhaps
`The [Realm] instance this object belongs to`

Reply:
Fixed